src/main/resources/init_db.sql

```diff
@@ -89,4 +89,6 @@ CREATE TABLE `users` (
 ) ENGINE=InnoDB AUTO_INCREMENT=8 DEFAULT CHARSET=utf8;
 LOCK TABLES `users` WRITE;
 INSERT INTO `users` VALUES (1,'Igor','Igor','123',1),(2,'Admin','Admin','123',0),(3,'Laci','Laci','123',1),(4,'Max','Max','123',0),(5,'Dima','Dima','123',0),(6,'Mika','Mika','123',0),(7,'Igor','Igor','123',1);
-UNLOCK TABLES;
\ No newline at end of file
+UNLOCK TABLES;
+ALTER TABLE `shop`.`users`
+ADD COLUMN `salt` VARBINARY(16) NULL AFTER `deleted`;
```

Comment:
I agree
alternatively, you can modify CREATE statement

by the way, what does NULL mean here?

Reply:
fixed null